Sources/LeafKit/Data/TemplateDataEncoder.swift

```diff
@@ -1,232 +0,0 @@
-// TODO: - Should this be reimplemented? This let Leaf3 use Futures as data types?
-
```

Comment:
Vapor/Leaf/LeafEncoder should probably be moved to LeafKit and would obviate reimplementing this file